riotfile.py

```diff
@@ -3,12 +3,41 @@
 global_deps = [
     "mock",
     "pytest<4",
-    "pytest-benchmark",
 ]
 
 global_env = [("PYTEST_ADDOPTS", "--color=yes")]
 
 suites = [
+    Suite(
+        name="black",
+        command="black --check .",
+        cases=[
+            Case(
+                pys=[3.8],
+                pkgs=[
+                    ("black", ["==20.8b1"]),
+                ],
+            ),
+        ],
+    ),
+    Suite(
+        name="flake8",
+        command="flake8 ddtrace/ tests/",
+        cases=[
+            Case(
+                pys=[3.8],
+                pkgs=[
+                    ("flake8", [">=3.8,<3.9"]),
+                    ("flake8-blind-except", [""]),
+                    ("flake8-builtins", [""]),
+                    ("flake8-docstrings", [""]),
+                    ("flake8-logging-format", [""]),
+                    ("flake8-rst-docstrings", [""]),
+                    ("pygments", [""]),
```

Comment:
```suggestion
                    ("flake8", [">=3.8,<3.9"]),
                    ("flake8-blind-except", ),
                    ("flake8-builtins", ),
                    ("flake8-docstrings", ),
                    ("flake8-logging-format", ),
                    ("flake8-rst-docstrings", ),
                    ("pygments", ),
```

is this more legible?

this comment has nothing to do with this PR

Reply:
We could also just have it be `[("flake8", ">=3.8,<3.9"), "flake8-blind-except", "flake8-builtins", "flake8-docstrings"]`.

Reply:
yeah